src/apps/frontend/types/task.ts

```diff
@@ -0,0 +1,24 @@
+import { JsonObject } from './common-types';
+
+export interface UpdateTaskParams {
+  accountId: string;
+  description: string;
+  taskId: string;
+  title: string;
+}
+export interface DeleteTaskParams {
```

Comment:
Are we using this `DeleteTaskParams` interface somewhere, if not then this can be removed.